grails-app/conf/spring/resources.groovy

```diff
@@ -1,5 +1,9 @@
 package spring
 
+import org.pih.warehouse.inboundSortation.CrossDockingStrategy
```

Comment:
Let's make this `inbound` for now. I'd like to come up with a better package structure, but I think inbound is good enough.